config/puma.rb

```diff
@@ -1,15 +1,15 @@
 # Puma can serve each request in a thread from an internal thread pool.
-# The `threads` method setting takes two numbers a minimum and maximum.
+# The `threads` method setting takes two numbers: a minimum and maximum.
 # Any libraries that use thread pools should be configured to match
 # the maximum value specified for Puma. Default is set to 5 threads for minimum
-# and maximum, this matches the default thread size of Active Record.
+# and maximum; this matches the default thread size of Active Record.
 #
-threads_count = ENV.fetch("RAILS_MAX_THREADS") { 5 }.to_i
+threads_count = ENV.fetch("RAILS_MAX_THREADS") { 5 }
```

Comment:
...why isn't .to_i necessary? It should handle weird env variables.

Reply:
It was removed in the default configuration. Do we need to handle weird env variables?

I tried looking for where the default configuration comes from but couldn't find it in the puma gem, so I couldn't see their rationale for removing it.